experimental/InvariantTheory/src/InvariantTheory.jl

```diff
@@ -475,7 +520,7 @@ function inv_generators(I::MPolyIdeal, G::LinearlyReductiveGroup, ringg::MPolyRi
     #we need the representation matrix and determinant of group matrix to be in basering(I)
     mixed_ring_xy = parent(genss[1])
     R = base_ring(M)
-    m = G.group[2]
+    m = group_dim(G)
```

Comment:
Can't you again eliminate `m` here and also below (there by using `end`)?

In any case I would use `xyz = gens(mixed_ring_xy)` to avoid the repeated calls to `gens`

Reply:
m is required in this function as it is used in the Reynolds operator.